cmd/ingest.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"go/types"
+
+	_ "github.com/lib/pq"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/network"
+	"github.com/stellar/go/support/config"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/wallet-backend/internal/ingest"
+)
+
+type ingestCmd struct{}
+
+func (c *ingestCmd) Command() *cobra.Command {
+	cfg := ingest.Configs{}
+	cfgOpts := config.ConfigOptions{
+		{
+			Name:        "database-url",
+			Usage:       "Database connection URL.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.DatabaseURL,
+			FlagDefault: "postgres://postgres@localhost:5432/wallet-backend?sslmode=disable",
+			Required:    true,
+		},
+		{
+			Name:        "network-passphrase",
+			Usage:       "Stellar Network Passphrase to connect.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.NetworkPassphrase,
+			FlagDefault: network.TestNetworkPassphrase,
+			Required:    true,
+		},
+		{
+			Name:        "captive-core-bin-path",
+			Usage:       "Path to Captive Core's binary file.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.CaptiveCoreBinPath,
+			FlagDefault: "/usr/local/bin/stellar-core",
```

Comment:
I wonder if this one couldn't also point to a local directory like the `captive-core-config-dir`.

Reply:
Whenever you install captive core locally using [these instructions](https://github.com/stellar/stellar-core/blob/master/INSTALL.md#macos), `/usr/local/bin/stellar-core` will be the default install location. If we wanted it to live inside a project directory we would need to change that install path.
I thought it was easier to use the default location and set it here accordingly. What do you think?